tutorials/plugins/gdnative/gdnative-cpp-example.rst

```diff
@@ -222,7 +316,7 @@ This one should be straightforward. We're implementing each method of
 our class that we defined in our header file.
 Note that the ``register_method`` call **must** expose the ``_process`` method,
 otherwise Godot will not be able to use it. However, we do not have to tell Godot
-about our constructor and destructor.
+about our constructor, destructor and ``_init`` functions.
```

Comment:
I think @cbscribe would request an Oxford comma here after "destructor".

Reply:
Or not sure actually here. Reads better without it for me actually to bind all three to the "functions".